src/main/java/com/couchbase/client/dcp/conductor/OperationTimeoutException.java

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright (c) 2016 Couchbase, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.couchbase.client.dcp.conductor;
+
+import com.couchbase.client.core.CouchbaseException;
+
+public class OperationTimeoutException extends CouchbaseException {
```

Comment:
ChannelTimeoutException so that it indicates it has something to do with the channel? Operation is very generic